core/src/main/java/feast/core/model/FeatureSet.java

```diff
@@ -262,6 +273,10 @@ public void addFeature(Feature feature) {
     return FeatureSetProto.FeatureSet.newBuilder().setMeta(meta).setSpec(spec).build();
   }
 
+  public Map<String, String> getLabelsMap() {
+    return TypeConversion.convertJsonStringToMap(this.labels);
```

Comment:
should this be `this.getLabels()`?

Reply:
There's a current helper function that converts this, used in a similar way [here](https://github.com/feast-dev/feast/blob/master/core/src/main/java/feast/core/model/Feature.java#L241). Should I avoid writing it that way?

Reply:
yea you should try to use methods as much as possible because you can add logic there later.